docarray/array/storage/pqlite/backend.py

```diff
@@ -20,7 +23,7 @@
 class PqliteConfig:
     n_dim: int = 1
```

Comment:
this pattern is not supported in weaviate and I guess the same reason applies for pqlite
Actually, implicitly setting vectors of dim=1 by default is problematic, because users will try to add empty Documents and then find errors when they want to update embeddings.
Instead, what we should do is make n_dim a positional argument and raise an error when no config object is passed
What do you think ?

Reply:
@alaeddine-13 Agree, I have fixed it, please give check!